package.json

```diff
@@ -9,7 +9,7 @@
     "theme": "dark"
   },
   "license": "Apache-2.0",
-  "publisher": "ivangsa",
+  "publisher": "asyncapi",
```

Comment:
I think this is the only thing that needs to be changed for marketplace, right @ivangsa ?

Reply:
yes, I think is the only change required..